dev/ci/scripts/utils/ci_utils.sh

```diff
@@ -189,12 +198,31 @@ function cleanup_experiment() {
 
 function build_compute () {
```

Comment:
I think we should just call this function `build`, because it is not building on compute nodes on Gaea.  The function internally knows which machines to build on compute vs serially.

Reply:
The Gaea C6 head node build is temporary work abound that should go away soon.